rules/assembly_hub.smk

```diff
@@ -63,5 +64,5 @@ rule bam_bigwig:
         deeptools_limit=1
     shell:
         """
-        bamCoverage --bam {input} --outFileName {output} --numberOfProcessors {threads} {params} --verbose >> {log} 2>&1
+        bamCoverage --bam {input.bam} --outFileName {output} --numberOfProcessors {threads} {params} --verbose >> {log} 2>&1
         """
```

Comment:
.bai file not used if im correct?

Reply:
not in the command line, but the script fails if it is not present.